modules/schema/pages/procedure-lot-part-information.adoc

```diff
@@ -2461,7 +2461,7 @@ This term is used to specify that any tenderer must submit tenders for
 all Lots. This element shall be used when and only when tendering for
 all existing Lots is mandatory. For this, the
 "_cbc:PartPresentationCode_" element is used with a single possible
-value: "_all_"
+value: "_all_".
```

Comment:
May be an issue, I think we have "**A**ll" in the codelist

Reply:
No the tenderlot-presentation.gc codelist has the lower-case code "all".